### DISCUSSION TITLE
Determine scope of Emissions factors (I) for Alpha data set

### DISCUSSION THREAD

**Discussion by dtoakley** [Green-Software-Foundation/software_carbon_intensity_data#6] [General]

**Comment by mrchrisadams** [comment]
I have a question here about the SCI model, and the data it relies on.

## emissions factors for electricity

As I understand it, the SCI uses a consequential, not an attributional model for working out the CO2 emissions of say… a kilowatt hour of electricity.

This means that I would need emissions intensity data that would cover the consequences of using a kilowatt hour of electricity at a given time, compared to doing so at another time, or in another place - as in the marginal intensity figures.

I know that there is a free marginal indensity _index_ API from Watttime, which you could plausibly use to get an _idea_ for forward looking decisions - this gives a relative number in the form of an index, not an absolute figure like 300g of CO2e per hour. You can see a demonstration of using the API below:

https://nextjournal.com/greenweb/experiments-with-the-free-marginal-carbon-intensity-from-wattime

This might still useful for making decisions about scheduling work (so, moving through time), but I think the SCI expects absolute measurements, and I wouldn't be able to use that information to make a decision about _where_ I should run it (i.e. moving through space), as I don't have an easy way to compare one region with another given.

Put another way, I can only compare their indices which are relative to _their own_ carbon intensity, not another region:

> the reported index value is given as a percentage between the lowest and highest MOER values observed in the past two weeks in the given grid region. A higher index indicates a higher MOE value (i.e. higher carbon intensity).

### Looking at space, not just time

If you wanted that, one option might be to have some figures in absolute terms - this would at least allow you to make a decision about _where_ to run a compute job, or even get an idea of the consequences or running it in one place versus or another (we might refer to these as marginal intensity figures).

Right now, finding emission factor figures that are open data is incredibly difficult, and the providers that might offer it also have business selling higher resolution (in both time and space) that they understandably would want to protect.

If you were to reduce the resolution in terms of time, annualised CO2 per kilowatt hour figures, in absolute terms feel like they would be a minimum level of information you might need to make a judgement about _where_ you ran something, as opposed to _when_ you ran a job.

You might even be able to compare them with an already free API like the marginal operating index, to get some intuition for time *and* space - this would allow folks to experiment along both dimensions, and get some idea of what is effective, while leaving room for licensing data if the extra precision turned to to be relevant for the use case.

@Henry-WattTime - I know you folks offer things like Marginal Operating Emissions index, but I'm not aware of any freely available and licensed figures for the annualised emissions data. Do you know any? And if this characterisation isn't quite right, would you let me know?

### Other efforts in this field

It's worth being aware of this campaign mentioned at the link below to get the IEA to release some of the information we might be intending to compile too:

> The International [Energy](https://www.theguardian.com/environment/energy) Agency is facing calls to make the national energy data it collects from governments publicly available.
> 
> This would aid independent research, which in turn could help to accelerate the global transition to low-carbon energy.
> 
> More than 30 international academics have written to the global energy watchdog to call for it to drop its paywalls for national energy datasets, which are collected using public funds, to avoid making climate action “more costly and less effective”.

Source: [Energy watchdog urged to give free access to government data](https://www.theguardian.com/environment/2021/dec/10/academics-urge-iea-to-give-free-access-to-national-energy-data) by @guardian

**Reply by Henry-WattTime** [reply]
The most easily accessible annual marginal emissions data I am aware of is published by the UN. This should provide a minimum backstop for annual data and allow regional comparisons. 
https://unfccc.int/climate-action/sectoral-engagement/ifis-harmonization-of-standards-for-ghg-accounting/ifi-twg-list-of-methodologies
https://unfccc.int/sites/default/files/resource/Harmonized_Grid_Emission_factor_data_set.xlsx

We are also in the process of getting the GSF access to the WattTime marginal emissions data. This will enable more granular assessment and decisions making.

**Reply by mrchrisadams** [reply]
It would be really great to be able to use this data. 

For Europe and America, there are two annual marginal figures which are surprisingly close too.

Country / Territory / Island | Firm Energy (e.g., Hydro, Geothermal)  | Intermittent Energy (e.g., Solar, Wind, Tidal)| Energy Efficiency   |  Electricity Consumption |  Operating Margin Grid Emission Factor, gCO2/kWh (including for use in PCAF GHG accounting) 
-- | -- | -- | -- | -- | --
United States | 246 | 352 | 246 | 246 | 416
Eu 27 | 261 | 353 | 261 | 261 | 409

I've looked into this it's not clear what the terms of licensing are for the data. I've spend a bit if time looking into this, as we've been working to creating a binary written in golang to expose these numbers. You can see the issue below for more notes as I investigate:

https://github.com/thegreenwebfoundation/grid-intensity-go/issues/8

I've also asked in the OpenMod forum, a community of energy modellers about the licensing, and shared what I learned when I emailed the webmaster - and their responses:

https://forum.openmod.org/t/open-letter-to-iea-and-member-countries-requesting-open-data/2949/14

@Henry-WattTime  - I've added a figure above for the US and Europe from these datasets, but it's not clear to me if you could end up with a global marginal figure for carbon intensity of electricity. Do people use global marginal intensity figures anywhere?
